scripts/run.sh

```diff
@@ -79,58 +71,30 @@ function updateLetsEncrypt() {
 
 function updateDatabase() {
     pullSetup
-    if [ $OS == "lin" ]
-    then
-        docker run -i --rm --name setup --network container:bitwarden-mssql \
-            -v $OUTPUT_DIR:/bitwarden -e $LUID -e $LGID bitwarden/setup:$COREVERSION \
-            dotnet Setup.dll -update 1 -db 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
-    else
-        docker run -i --rm --name setup --network container:bitwarden-mssql \
-            -v $OUTPUT_DIR:/bitwarden bitwarden/setup:$COREVERSION \
-            dotnet Setup.dll -update 1 -db 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
-    fi
+    docker run -i --rm --name setup --network container:bitwarden-mssql \
+        -v $OUTPUT_DIR:/bitwarden --env-file $ENV_DIR/uid.env bitwarden/setup:$COREVERSION \
+        dotnet Setup.dll -update 1 -db 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
     echo "Database update complete"
 }
 
 function update() {
     pullSetup
-    if [ $OS == "lin" ]
-    then
-        docker run -i --rm --name setup -v $OUTPUT_DIR:/bitwarden \
-            -e $LUID -e $LGID bitwarden/setup:$COREVERSION \
-            dotnet Setup.dll -update 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
-    else
-        docker run -i --rm --name setup \
-            -v $OUTPUT_DIR:/bitwarden bitwarden/setup:$COREVERSION \
-            dotnet Setup.dll -update 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
-    fi
+    docker run -i --rm --name setup -v $OUTPUT_DIR:/bitwarden \
+        --env-file $ENV_DIR/uid.env bitwarden/setup:$COREVERSION \
+        dotnet Setup.dll -update 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
 }
 
 function printEnvironment() {
     pullSetup
-    if [ $OS == "lin" ]
-    then
-        docker run -i --rm --name setup -v $OUTPUT_DIR:/bitwarden \
-            -e $LUID -e $LGID bitwarden/setup:$COREVERSION \
-            dotnet Setup.dll -printenv 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
-    else
-        docker run -i --rm --name setup \
-            -v $OUTPUT_DIR:/bitwarden bitwarden/setup:$COREVERSION \
-            dotnet Setup.dll -printenv 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
-    fi
+    docker run -i --rm --name setup -v $OUTPUT_DIR:/bitwarden \
+        --env-file $ENV_DIR/uid.env bitwarden/setup:$COREVERSION \
+        dotnet Setup.dll -printenv 1 -os $OS -corev $COREVERSION -webv $WEBVERSION
 }
 
 function restart() {
     dockerComposeDown
     dockerComposePull
     updateLetsEncrypt
-    
-    if [ $OS == "lin" ]
```

Comment:
Why was this removed? User can change between restarts. ex. going from root to non-root user.

Reply:
`$LOCAL_UID` and `$LOCAL_GID` are set and saved at installation time in `uid.env`, to the UID/GID of the user performing the installation.
At run time, `uid.env` is loaded, and the UID/GID are used by `entrypoint.sh` files to run the services, but `65534/65534` if saved UID/GID are set to 0 (root).

So if `kyle/kyle` performed the installation, subsequent starts will use `kyle/kyle` to run the services.
If `root` performed the installation, `65534/65534` (`nobody/nogroup`) will be used by default, for security reasons (we don't want services to run as `root` user).

User could also want and manually update the `uid.env` file to switch services to another user.
Think about `root` installing the application. He can then modify the `uid.env` file to switch to whatever service user he has properly configured on the system to run Bitwarden with.
Subsequent starts will then use this specific service user, even if Bitwarden is started from root, at boot time etc...
This is how services generally work on UNIX / Linux.

Overwriting `uid.env` each time application starts is useless, and prevents for example `root` to define a specific service user.

Reply:
This will now break if the user is upgrading from a version where `uid.env` did not exist. This functionality was only recently added and these scripts must remain backwards compatible.

Reply:
You're right, good catch :
`ERROR: Couldn't find env file: /home/bw/bwdata/env/uid.env`
Let me add the required check / few lines 👍

Reply:
This does the trick :
https://github.com/bitwarden/core/pull/292/commits/30721b7a8ea32d6cd79119339fa3630a59334214